### ISSUE TITLE
Ambient Sound from Imported Soundset not playing / found

### ISSUE THREAD

**Issue by Truncated** [frondeus/fvtt-syrin-control#48]
**Describe the bug**

When an ambient sound is made from an imported soundset sound, the ambient audio item triggers but the psuedo-audio .wav path "syrin-scape-not-a-path.wav" returns a 404 error, so nothing is played.

On load, recieves error of : 
GET https://localhost:23232/syrinscape-not-a-real-path.wav 404 (Not Found)
Foundry VTT | Failed to load audio node
![image](https://user-images.githubusercontent.com/14208325/194941474-d7bccd54-5772-4b2f-8ad9-e8e4aa093a12.png)

Manually playing it from the Audio Playlists works as expected; connection does function otherwise.

**To Reproduce**
1. Open Soundscape Online Player (windows x64 install version)
2. Import a custom Soundset
3. Drag the Sound onto the map to create an Ambient sound from it
4. Move token near sound icon.

**Expected behavior**
When the player token moved into the borders of the sound circle, it should start hearing the audio at a low volume. This volume should climb to the configured maximum as the token moved closer to the ambient audio center point.

**Screenshots**
Playing manually
![image](https://user-images.githubusercontent.com/14208325/194937439-abb6cdb1-c847-46e0-959e-d55b02bd3898.png)

Stopped manually, then went to player near ambiant sound... nothing, online player never gets a message.
![image](https://user-images.githubusercontent.com/14208325/194937540-83f195e1-790e-409e-ae4d-3ba26100ca53.png)

Tried Find the Culprit...
![image](https://user-images.githubusercontent.com/14208325/194938648-84990a3c-f19c-47f0-b0e8-e4d7bcddba50.png)

**Details**
 - OS: Win 10
 - Browser Chrome (logged in as GM directly to Foundry server, running locally)
![image](https://user-images.githubusercontent.com/14208325/194936913-b88d4d99-43d1-4cfa-9cfe-e3e408a46e41.png)

 - Module version: 0.4.0-alpha-3
 - [X] Does it work in Safety Mode (without other plugins enabled)  - No, it does not.
 - [x] I did read FAQ - https://github.com/frondeus/fvtt-syrin-control/wiki/Frequently-Asked-Questions

**Logs**
```
​ -90
​ 75.96375653207355
​ 83.37922391768501
​ -84.06258390051846
​ 56.65929265352301
​ -9.833563964207116
​ 11.309932474020215
​ -17.525568373722876
​ 0
​ -90
​ -4.96974072811031
​ 0
​ 83.15722658736905
​ -90
​ -83.15722658736905
2​ -90
​ 85.76360520094116
​ -90
​ 79.04593735660166
​ 63.434948822922024
​ 52.76516601842533
​ 26.56505117707799
​ 14.57421619803874
​ 0
​ -11.842076115327771
​ -36.501441120506314
​ -56.659292653523
​ -63.43494882292201
​ -78.15792388467223
​ 80.90972307917767
​ 56.30993247402023
​ 29.604450746004915
​ -158.355564859286
​ -18.434948822921996
​ -56.3099324740202
​ -30.827662299868024
​ 80.90972307917767
​ 58.902687114220555
​ -90
​ 0
​ -90
​ 0
​ 6.115503566285412
​ 0
​ -60.113473059575966
​ 0
​ 18.43494882292201
​ -21.96056402009647
​ -97.76516601842535
​ -64.35899417569472
​ -25.64100582430528
​ -5.381547052131921
​ 64.3589941756947
​ -72.47443162627712
​ 79.04593735660166
​ -64.3589941756947
​ 56.65929265352301
​ 81.32659750118401
​ -11.659292653522982
​ -165.42578380196127
​ -6.62077608231499
​ 0
​ -164.1406337333943
​ -6.3401917459099195
​ -72.03086025984297
​ 80.90972307917767
​ 0
​ -158.355564859286
​ -135
​ 0
3​ -90
2​ 0
​ -45
​ -70.64100582430527
​ 81.59662890943918
​ -44.236101539070006
​ 0
​ -79.2157021324374
​ -10.341645294478894
​ 0
4​ -90
​ 0
​ -80.16643603579286
​ -56.3099324740202
​ -29.886526940424005
​ -13.495733280795775
​ 13.495733280795804
2​ -90
​ 80.90972307917767
​ -54.83356396420709
​ 0
​ -64.3589941756947
​ -18.434948822921996
​ 6.6207760823149755
​ -83.15722658736905
​ 0
​ -96.06178878872865
​ -45
​ 44.236101539070006
​ -45
2​ -134.23610153907
​ 44.236101539070006
2​ -45
​ -134.23610153907
​ 44.236101539070006
2​ -45
​ -134.23610153907
​ 44.236101539070006
2​ -45
​ -134.23610153907
​ 44.236101539070006
2​ -45
​ -134.23610153907
​ 44.236101539070006
2​ -45
​ -134.23610153907
​ 44.236101539070006
2​ -45
​ -134.23610153907
​ 44.236101539070006
2​ -45
​ -54.09027692082233
​ 75.42578380196124
​ 37.23483398157467
​ 0
​ -45
​ -116.56505117707799
​ -45
​ -117.47443162627712
​ -135
​ -115.64100582430528
​ -45
​ -154.3589941756947
​ -79.04593735660167
​ 41.34777721969368
​ -90
​ -76.5042667192042
​ -10.7842978675626
​ -90
​ 75.42578380196124
​ 13.495733280795804
​ 45.763898460929994
​ -56.659292653523
​ -23.96248897457818
​ -66.86179888188792
​ 55.819883169416556
​ -112.16634582208246
​ 27.474431626277138
​ -64.05770451012833
​ -157.61986494804043
​ -27.474431626277138
​ -23.96248897457818
​ -66.86179888188792
​ 55.819883169416556
​ -112.16634582208246
​ 27.474431626277138
​ -64.05770451012833
​ -157.61986494804043
​ -27.474431626277138
8​ -90
2​ 0
​ -90
​ 0
2​ -90
​ -4.573921259900857
​ -67.19545952867895
​ 8.240403290564785
​ -23.96248897457818
​ 55.819883169416556
​ -112.16634582208246
​ 27.474431626277138
​ -157.61986494804043
​ 55.819883169416556
​ 27.474431626277138
​ -27.474431626277138
2​ -66.86179888188792
​ -64.05770451012833
​ -23.96248897457818
​ -64.05770451012833
​ -157.61986494804043
​ -112.16634582208246
​ -27.474431626277138
​ 0
​ Safety Tools | Deactivated
​ DEBUG | Calling getSceneControlButtons hook with args:
​ Array(1)
​ SelectToolEverywhere | Tools added.
​ Safety Tools | Adding scene controls
​ Token Attacher | Tools added.
​ DEBUG | Calling libWrapper.Register hook with args:
​ Array(5)
​ DEBUG | Calling libWrapper.Register hook with args:
​ Array(5)
​ DEBUG | Calling libWrapper.Register hook with args:
​ Array(5)
​ DEBUG | Calling libWrapper.Register hook with args:
​ Array(5)
​ DEBUG | Calling libWrapper.Register hook with args:
​ Array(5)
​ DEBUG | Calling libWrapper.Register hook with args:
​ Array(5)
​ DEBUG | Calling libWrapper.Register hook with args:
​ Array(5)
​ DEBUG | Calling libWrapper.Register hook with args:
​ Array(5)
​ DEBUG | Calling libWrapper.Register hook with args:
​ Array(5)
​ DEBUG | Calling libWrapper.Register hook with args:
​ Array(5)
​ DEBUG | Calling libWrapper.Register hook with args:
​ Array(5)
​ DEBUG | Calling libWrapper.Register hook with args:
​ Array(5)
​ DEBUG | Calling diceSoNiceInit hook with args:
​ Array(1)
​ DEBUG | Calling getMonksCombatTrackerEntryContext hook with args:
​ Array(2)
​ DEBUG | Calling getCombatTrackerEntryContext hook with args:
​ Array(2)
​ DEBUG | Calling getSidebarTabEntryContext hook with args:
​ Array(2)
​ DEBUG | Calling renderMonksCombatTracker hook with args:
​ Array(3)
​ DEBUG | Calling renderCombatTracker hook with args:
​ Array(3)
​ update scene
​ DEBUG | Calling renderSidebarTab hook with args:
​ Array(3)
​ DEBUG | Calling renderSceneControls hook with args:
​ Array(3)
​ DEBUG | Calling renderApplication hook with args:
​ Array(3)
​ DEBUG | Calling getSceneDirectoryFolderContext hook with args:
​ Array(2)
​ DEBUG | Calling getSidebarDirectoryFolderContext hook with args:
​ Array(2)
​ DEBUG | Calling getSidebarTabFolderContext hook with args:
​ Array(2)
​ DEBUG | Calling getSceneDirectoryEntryContext hook with args:
​ Array(2)
​ DEBUG | Calling getSidebarDirectoryEntryContext hook with args:
​ Array(2)
​ DEBUG | Calling getSidebarTabEntryContext hook with args:
​ Array(2)
​ DEBUG | Calling renderSceneDirectory hook with args:
​ Array(3)
​ DEBUG | Calling renderSidebarDirectory hook with args:
​ Array(3)
​ DEBUG | Calling renderSidebarTab hook with args:
​ Array(3)
​ DEBUG | Calling pingsReady hook with args:
​ Array(1)
​ DEBUG | Calling simple-calendar-primary-gm hook with args:
​ Array(1)
​ DEBUG | Calling simple-calendar-ready hook with args:
​ Array(1)
​ Compendium Folders | Cleared cached folder structure
​ DEBUG | Calling addedAllQuestEntries hook with args:
​ Array(0)
​ DEBUG | Calling ForienQuestLog.Lifecycle.ready hook with args:
​ Array(0)
​ DEBUG | Calling updateScene hook with args:
​ Array(4)
​ SyrinControl | Syrin | Set Active Mood
trace @ index.js:10305
setActiveMood @ index.js:10771
(anonymous) @ index.js:14577
_call @ foundry.js:294
callAll @ foundry.js:253
callback @ foundry.js:10313
(anonymous) @ foundry.js:10251
_handleUpdateDocuments @ foundry.js:10251
_updateDocuments @ foundry.js:10131
​ DEBUG | Calling getSceneNavigationContext hook with args:
​ Array(2)
​ DEBUG | Calling renderSceneNavigation hook with args:
​ Array(3)
​ DEBUG | Calling renderApplication hook with args:
​ Array(3)
constants.js:96 Uncaught (in promise) TypeError: Cannot convert undefined or null to object
[Detected 1 package: token-says]
    at Function.entries (<anonymous>)
    at _determineWorldOptions (constants.js:96)
    at index.js:294
    at Function._call (foundry.js:294)
    at Function.callAll (foundry.js:253)
    at Game.setupGame (foundry.js:5567)
    at async Game._initializeGameView (foundry.js:6514)
    at async Game.initialize (foundry.js:5431)
_determineWorldOptions @ constants.js:96
(anonymous) @ index.js:294
_call @ foundry.js:294
callAll @ foundry.js:253
setupGame @ foundry.js:5567
​ Foundry VTT | Rendering TokenActionHUD
​ Sequencer | Ready to go!
​ DEBUG | Calling sequencer.ready hook with args:
​ Array(0)
​ Sequencer | Database | Entries for "jb2a" registered
​ DEBUG | Calling sequencerReady hook with args:
​ Array(0)
​ DEBUG | Calling DirectorInitIntegrations hook with args:
​ Array(0)
​ director:info Init Director integrations
​ DEBUG | Calling sequencerEffectManagerReady hook with args:
​ Array(0)
​ director:info update current scene: Jzadirune
​ director:info true Array(0)
​ DEBUG | Calling DirectorUpdateSequences hook with args:
​ Array(1)
​ director:info Started!
​ DEBUG | Calling DirectorUpdateSequences hook with args:
​ Array(1)
​ DEBUG | Calling DirectorUpdateActions hook with args:
​ Array(1)
​ DEBUG | Calling DirectorUpdateActions hook with args:
​ Array(1)
​ DEBUG | Calling DirectorUpdateHooks hook with args:
​ Array(1)
​ Scrolling from 10249 to 10304
​ -90
​ 0
2​ -90
​ 0
13​ -90
​ 0
3​ -90
​ 0
​ -90
5​ 0
​ -90
​ 0
​ -90
4​ 0
3​ -90
​ 0
4​ -90
3​ 0
​ -90
2​ 0
​ -90
​ 0
​ -90
​ -104.57421619803876
​ -90
​ -71.11391263029071
​ -45
​ 14.57421619803874
​ -7.406912128495236
​ 5.420595283762367
2​ 0
​ -6.562698508937473
​ 5.93741609948151
​ -1.8072769120298062
​ 8.719746506237328
​ -9.090276920822319
​ 6.842773412630947
​ -11.309932474020215
​ 21.96056402009647
​ -6.842773412630947
​ 0
​ -18.886087369709287
​ -55.95406264339834
​ -95.93741609948152
​ -75.42578380196126
​ 0
​ 21.96056402009647
​ 0
​ 64.3589941756947
​ -90
​ 62.52556837372288
​ -75.42578380196124
​ -90
​ 0
​ -90
​ -95.0058325529511
​ -68.355564859286
​ -99.83356396420713
​ -90
​ -86.69325351975748
​ -90
​ -94.95325747784207
​ -84.06258390051849
​ -90
​ -94.2484545293613
​ 0
​ -7.406912128495264
​ 0
​ -108.43494882292202
​ -90
2​ 0
​ -90
​ 0
​ -90
​ 0
​ -90
​ 64.3589941756947
​ -90
2​ 0
​ -90
​ -108.8860873697093
​ -10.7842978675626
​ 34.045937356601655
2​ -90
​ -84.51640955553555
​ 87.25191181994626
​ -64.3589941756947
​ -170.90972307917767
​ 0
​ -45
2​ -90
​ -27.474431626277124
2​ 0
​ -177.17019240711284
2​ 0
​ -2.127151112850811
​ -174.57940471623766
​ -14.036243467926482
​ -154.3589941756947
​ 0
​ 85.47404087774305
​ 75.42578380196124
2​ 0
​ 62.52556837372288
2​ -90
​ 4.236394799058843
​ 0
​ 27.474431626277138
​ 75.42578380196124
​ 83.88449643371462
​ -59.57421619803873
​ 0
​ -6.115503566285383
​ -116.56505117707799
​ -99.83356396420713
​ -154.3589941756947
​ -5.005832552951119
​ -8.403371090560825
​ -165.42578380196127
​ -90
​ -62.52556837372287
​ -137.29061004263855
​ -152.52556837372288
​ -16.03233935393598
​ -5.937416099481538
​ 0
​ -25.641005824305296
2​ -90
​ 18.43494882292201
​ 0
​ -90
​ 0
​ -90
​ 0
6​ -90
​ 0
​ -90
​ 0
​ -90
​ 0
​ -90
2​ 0
​ -73.96766064606402
​ 84.51640955553557
​ -76.50426671920417
​ 0
​ -6.842773412630947
​ 0
​ 9.462322208025611
​ -7.614711942031022
​ 5.4300510730119385
​ -14.57421619803874
​ -99.83356396420713
​ -99.09027692082233
​ -51.115503566285405
​ 17.96913974015702
​ 0
​ 45
​ -3.509026588616564
​ -51.115503566285405
​ -21.644435140713995
​ 18.43494882292201
​ 45
​ 82.14668669802177
​ -75.96375653207352
​ 0
​ -173.15722658736905
​ -6.115503566285383
​ -173.93821121127138
​ 0
​ 80.16643603579288
​ -75.42578380196124
​ 78.15792388467224
​ -45
​ -8.403371090560825
​ -176.02326902492243
​ 0
​ -178.22853025996594
​ 0
​ -175.04674252215793
​ -126.50144112050633
​ -82.14668669802178
​ -78.340707346477
​ -99.83356396420713
​ 26.381411486958214
​ -73.09072818945918
​ 0
​ -90
​ 0
​ -90
​ 0
4​ -90
​ 0
​ -90
​ 0
​ -90
​ 0
3​ -90
​ 0
​ -90
2​ 0
​ -90
2​ 0
4​ -90
2​ 0
​ -153.434948822922
​ -4.953257477842044
​ -13.495733280795775
​ -145.95406264339834
​ -14.036243467926482
​ -154.3589941756947
2​ -90
​ -82.23483398157467
​ 0
​ -90
​ 0
​ -90
​ 0
​ -89.9181799035832
​ -14.014457246633413
​ -90
​ 21.644435140713995
​ -6.842773412630947
2​ 0
2​ -90
​ 0
2​ -90
​ -0.10520199288772858
​ -96.06178878872865
​ -73.96766064606403
​ -98.13010235415598
2​ -90
​ -80.53767779197439
​ -95.42059528376238
​ -62.52556837372287
​ -90
​ 75.96375653207355
​ 83.37922391768501
​ -84.06258390051846
​ 56.65929265352301
​ -9.833563964207116
​ 11.309932474020215
​ -17.525568373722876
​ 0
​ -90
​ -4.96974072811031
​ 0
​ 83.15722658736905
​ -90
​ -83.15722658736905
2​ -90
​ 85.76360520094116
​ -90
​ 79.04593735660166
​ 63.434948822922024
​ 52.76516601842533
​ 26.56505117707799
​ 14.57421619803874
​ 0
​ -11.842076115327771
​ -36.501441120506314
​ -56.659292653523
​ -63.43494882292201
​ -78.15792388467223
​ 80.90972307917767
​ 56.30993247402023
​ 29.604450746004915
​ -158.355564859286
​ -18.434948822921996
​ -56.3099324740202
​ -30.827662299868024
​ 80.90972307917767
​ 58.902687114220555
​ -90
​ 0
​ -90
​ 0
​ 6.115503566285412
​ 0
​ -60.113473059575966
​ 0
​ 18.43494882292201
​ -21.96056402009647
​ -97.76516601842535
​ -64.35899417569472
​ -25.64100582430528
​ -5.381547052131921
​ 64.3589941756947
​ -72.47443162627712
​ 79.04593735660166
​ -64.3589941756947
​ 56.65929265352301
​ 81.32659750118401
​ -11.659292653522982
​ -165.42578380196127
​ -6.62077608231499
​ 0
​ -164.1406337333943
​ -6.3401917459099195
​ -72.03086025984297
​ 80.90972307917767
​ 0
​ -158.355564859286
​ -135
​ 0
3​ -90
2​ 0
​ -45
​ -70.64100582430527
​ 81.59662890943918
​ -44.236101539070006
​ 0
​ -79.2157021324374
​ -10.341645294478894
​ 0
4​ -90
​ 0
​ -80.16643603579286
​ -56.3099324740202
​ -29.886526940424005
​ -13.495733280795775
​ 13.495733280795804
2​ -90
​ 80.90972307917767
​ -54.83356396420709
​ 0
​ -64.3589941756947
​ -18.434948822921996
​ 6.6207760823149755
​ -83.15722658736905
​ 0
​ -96.06178878872865
​ -45
​ 44.236101539070006
​ -45
2​ -134.23610153907
​ 44.236101539070006
2​ -45
​ -134.23610153907
​ 44.236101539070006
2​ -45
​ -134.23610153907
​ 44.236101539070006
2​ -45
​ -134.23610153907
​ 44.236101539070006
2​ -45
​ -134.23610153907
​ 44.236101539070006
2​ -45
​ -134.23610153907
​ 44.236101539070006
2​ -45
​ -134.23610153907
​ 44.236101539070006
2​ -45
​ -54.09027692082233
​ 75.42578380196124
​ 37.23483398157467
​ 0
​ -45
​ -116.56505117707799
​ -45
​ -117.47443162627712
​ -135
​ -115.64100582430528
​ -45
​ -154.3589941756947
​ -79.04593735660167
​ 41.34777721969368
​ -90
​ -76.5042667192042
​ -10.7842978675626
​ -90
​ 75.42578380196124
​ 13.495733280795804
​ 45.763898460929994
​ -56.659292653523
​ -23.96248897457818
​ -66.86179888188792
​ 55.819883169416556
​ -112.16634582208246
​ 27.474431626277138
​ -64.05770451012833
​ -157.61986494804043
​ -27.474431626277138
​ -23.96248897457818
​ -66.86179888188792
​ 55.819883169416556
​ -112.16634582208246
​ 27.474431626277138
​ -64.05770451012833
​ -157.61986494804043
​ -27.474431626277138
8​ -90
2​ 0
​ -90
​ 0
2​ -90
​ -4.573921259900857
​ -67.19545952867895
​ 8.240403290564785
​ -23.96248897457818
​ 55.819883169416556
​ -112.16634582208246
​ 27.474431626277138
​ -157.61986494804043
​ 55.819883169416556
​ 27.474431626277138
​ -27.474431626277138
2​ -66.86179888188792
​ -64.05770451012833
​ -23.96248897457818
​ -64.05770451012833
​ -157.61986494804043
​ -112.16634582208246
​ -27.474431626277138
​ 0
​ Safety Tools | Deactivated
​ DEBUG | Calling getSceneControlButtons hook with args:
​ Array(1)
​ SelectToolEverywhere | Tools added.
​ Safety Tools | Adding scene controls
​ Token Attacher | Tools added.
​ DEBUG | Calling renderSceneControls hook with args:
​ Array(3)
​ DEBUG | Calling renderApplication hook with args:
​ Array(3)
​ Foundry VTT | Retrieved and compiled template modules/data-inspector/template/object.hbs
​ Foundry VTT | Retrieved and compiled template modules/data-inspector/template/inspector.hbs
​ Foundry VTT | Retrieved and compiled template modules/data-inspector/template/value.hbs
​ Foundry VTT | Retrieved and compiled template modules/data-inspector/template/object-data.hbs
​ DATA INSPECTOR 🔍 | 3.1.0 | READY
​ Image retainer detected 0 patch files in imageretainer/pf1
​ Foundry VTT | Retrieved and compiled template modules/koboldworks-pf1-starter-bag/template/form.hbs
​ Foundry VTT | Retrieved and compiled template modules/koboldworks-pf1-starter-bag/template/form-listing.hbs
​ Foundry VTT | Retrieved and compiled template modules/mkah-pf1-buff-activator/buff-selector.hbs
​ Foundry VTT | Retrieved and compiled template modules/mkah-pf1-resource-drain/refund-card.hbs
​ Foundry VTT | Retrieved and compiled template modules/mkah-pf1-resource-drain/editor.hbs
​ Foundry VTT | Retrieved and compiled template templates/hud/dialog.html
​ DEBUG | Calling getDialogHeaderButtons hook with args:
​ Array(2)
​ DEBUG | Calling getApplicationHeaderButtons hook with args:
​ Array(2)
​ Foundry VTT | Constructed index of pf1.roll-tables Compendium containing 2 entries
​ Foundry VTT | Retrieved and compiled template modules/compendium-folders/templates/compendium-directory.html
​ DEBUG | Calling getCompendiumFolderDirectoryEntryContext hook with args:
​ Array(2)
​ DEBUG | Calling getCompendiumDirectoryEntryContext hook with args:
​ Array(2)
​ DEBUG | Calling getSidebarDirectoryEntryContext hook with args:
​ Array(2)
​ DEBUG | Calling getSidebarTabEntryContext hook with args:
​ Array(2)
​ DEBUG | Calling getCompendiumFolderDirectoryFolderContext hook with args:
foundry.js:274 Array(2)
foundry.js:273 DEBUG | Calling getSidebarDirectoryFolderContext hook with args:
foundry.js:274 Array(2)
foundry.js:273 DEBUG | Calling getSidebarTabFolderContext hook with args:
foundry.js:274 Array(2)
foundry.js:273 DEBUG | Calling getCompendiumFolderDirectoryEntryContext hook with args:
foundry.js:274 Array(2)
foundry.js:273 DEBUG | Calling getCompendiumDirectoryEntryContext hook with args:
foundry.js:274 Array(2)
foundry.js:273 DEBUG | Calling getSidebarDirectoryEntryContext hook with args:
foundry.js:274 Array(2)
foundry.js:273 DEBUG | Calling getSidebarTabEntryContext hook with args:
foundry.js:274 Array(2)
foundry.js:273 DEBUG | Calling renderCompendiumFolderDirectory hook with args:
foundry.js:274 Array(3)
foundry.js:273 DEBUG | Calling renderCompendiumDirectory hook with args:
foundry.js:274 Array(2)
foundry.js:273 DEBUG | Calling renderSidebarDirectory hook with args:
foundry.js:274 Array(3)
foundry.js:273 DEBUG | Calling renderSidebarTab hook with args:
foundry.js:274 Array(3)
foundry.js:247 DEBUG | Calling lightingRefresh hook with args:
foundry.js:248 Array(1)
foundry.js:4660 Foundry VTT | Retrieved and compiled template /modules/token-action-hud/templates/template.hbs
foundry.js:273 DEBUG | Calling renderTokenActionHUD hook with args:
foundry.js:274 Array(3)
foundry.js:273 DEBUG | Calling renderApplication hook with args:
foundry.js:274 Array(3)
foundry.js:4660 Foundry VTT | Retrieved and compiled template templates/app-window.html
foundry.js:273 DEBUG | Calling renderDialog hook with args:
foundry.js:274 Array(3)
foundry.js:273 DEBUG | Calling renderApplication hook with args:
foundry.js:274 Array(3)
foundry.js:11980 Foundry VTT | Constructed index of pf1.ultimate-equipment Compendium containing 227 entries
foundry.js:273 DEBUG | Calling diceSoNiceReady hook with args:
foundry.js:274 Array(1)
foundry.js:11980 Foundry VTT | Constructed index of backstory-cards.tables Compendium containing 1 entries
foundry.js:4660 Foundry VTT | Retrieved and compiled template modules/theripper-premium-hub/templates/announcements.hbs
foundry.js:11980 Foundry VTT | Constructed index of backstory-cards-assets-library.tables Compendium containing 1 entries
data-inspector.mjs:87 DATA INSPECTOR 🔍 | System specific extensions loaded.
foundry.js:11980 Foundry VTT | Constructed index of backstory-cards-expansion-1.tables Compendium containing 3 entries
foundry.js:11980 Foundry VTT | Constructed index of backstory-cards-expansion-2.tables Compendium containing 1 entries
foundry.js:11980 Foundry VTT | Constructed index of backstory-cards-expansion-3.tables Compendium containing 6 entries
foundry.js:11980 Foundry VTT | Constructed index of foundry_community_tables.community-tables-5e Compendium containing 1 entries
foundry.js:11980 Foundry VTT | Constructed index of foundry_community_tables.community-tables-independent Compendium containing 184 entries
foundry.js:11980 Foundry VTT | Constructed index of pf-content.pf-encounter-tables Compendium containing 51 entries
foundry.js:11980 Foundry VTT | Constructed index of pf-content.pf-magic-tables Compendium containing 314 entries
foundry.js:11980 Foundry VTT | Constructed index of pf-content.pf-tables Compendium containing 28 entries
foundry.js:11980 Foundry VTT | Constructed index of pf-content.kingdom-building-tables Compendium containing 5 entries
foundry.js:11980 Foundry VTT | Constructed index of token-mold.adjectives Compendium containing 7 entries
utils.ts:32 SyrinControl | RAW Headless | Syrinscape | audio context false Object
trace @ utils.ts:32
configure @ raw.ts:47
init @ headlessPlayer.e8c93ef681d700cdb560.js:157
utils.ts:32 SyrinControl | RAW Headless | Syrinscape | Init Configure Object
trace @ utils.ts:32
configure @ raw.ts:59
init @ headlessPlayer.e8c93ef681d700cdb560.js:157
utils.ts:32 SyrinControl | RAW Headless | Syrinscape | On Active Object
trace @ utils.ts:32
onActive @ raw.ts:72
init @ headlessPlayer.e8c93ef681d700cdb560.js:157
utils.ts:32 SyrinControl | API | Online Soundsets | soundsets =  Array(949)
trace @ utils.ts:32
onlineSoundsets @ api.ts:53
utils.ts:32 SyrinControl | API | Online Global Elements
trace @ utils.ts:32
onlineGlobalElements @ api.ts:75
(anonymous) @ main.ts:207
utils.ts:32 SyrinControl | RAW | Get Global Elements
trace @ utils.ts:32
getGlobalElements @ raw.ts:261
onlineGlobalElements @ api.ts:77
(anonymous) @ main.ts:207
utils.ts:32 SyrinControl | API | Online Global Elements | elements =  Array(47)
trace @ utils.ts:32
onlineGlobalElements @ api.ts:79
foundry.js:63214 Foundry VTT | Activating pending audio playback with user gesture.
syrinscape-not-a-real-path.wav:1 Failed to load resource: the server responded with a status of 404 (Not Found)
foundry.js:63419 Foundry VTT | Failed to load audio node: Event
_createNode @ foundry.js:63419
foundry.js:273 DEBUG | Calling dragEndDialog hook with args:
foundry.js:274 Array(1)
foundry.js:273 DEBUG | Calling closeDialog hook with args:
foundry.js:274 Array(2)
foundry.js:273 DEBUG | Calling closeApplication hook with args:
foundry.js:274 Array(2)
```

**Additional Context**
I have https enabled with a cert to a host name I own. The link generated here shows localhost, but I'm going to start testing with browsers logging in from different machines to see if the GM direct app window is the issue.

**Comment by frondeus** [comment]
Hi thanks for submitting the bug.

This is something currently I can't really solve. Syrinscape does not provide a way to play mood for single player. Meaning, if there are two tokens, each triggering different ambient sound there is no defined way to decide which mood should be played.

I will keep this issue open tho, because I should somehow prevent from creating such a ambient sound.

**Comment by frondeus** [comment]
Fixed in 0.4.0-beta2

For now SyrinControl will play only one audio sound - the one that is the loudest - for all players.
